cmd/scheduler-estimator/app/options/options.go

```diff
@@ -42,7 +42,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	if o == nil {
 		return
 	}
-	fs.StringVar(&o.KubeConfig, "kubeconfig", o.KubeConfig, "Path to karmada control plane kubeconfig file.")
+	fs.StringVar(&o.KubeConfig, "kubeconfig", o.KubeConfig, "Path to member cluster's kubeconfig file.")
```

Comment:
I see there are two kinds of flag: `kubeconfig` and `karmada-kubeconfig`. `kubeconfig` make users confused(not sure its member clusters' kubeconfig to control plane's kubeconfig), so how about change the name, like `member-kubeconfig`? cc @lonelyCZ @XiShanYongYe-Chang 

And also, I see some components use `kubeconfig` as the control plane's kubeconfig: https://github.com/karmada-io/karmada/blob/07bb65bfe7990c3ff727640907f81e25615b4523/cmd/descheduler/app/options/options.go#L78, not unified.

Reply:
> kubeconfig make users confused(not sure its member clusters' kubeconfig or control plane's kubeconfig),

Agree. But it is out of this PR's scope.

In addition, if we are going to rename the flag, we should deprecate the old one.